hyperspyui/desktop_integration.py

```diff
@@ -1,175 +1,22 @@
-# -*- coding: utf-8 -*-
-# Copyright 2014-2016 The HyperSpyUI developers
-#
-# This file is part of HyperSpyUI.
-#
-# HyperSpyUI is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# HyperSpyUI is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with HyperSpyUI.  If not, see <http://www.gnu.org/licenses/>.
-"""
-Created on Sun Apr 12 17:43:47 2015
-
-@author: Vidar Tonaas Fauske
-"""
-
 import sys
-import os
-import platform
-import subprocess
-
-
 import argparse
 
 parser = argparse.ArgumentParser(
     description='Install HyperspyUI into the OS')
 parser.add_argument('-remove', action="store_true", help='uninstall flag')
 parser.add_argument('-no-shortcuts', action="store_true",
```

Comment:
Not really sure about this, but I think that `--` is more usual than `-` (which is usually for single letter argument)?

Reply:
I've seen both styles in BASH commands but I agree that ``-`` is most often reserved to single letter args. I'll change it.